oeps/oep-0045/decisions/0001-tutor-as-replacement-for-edx-configuration.rst

```diff
@@ -0,0 +1,499 @@
+is a couple of months behind the latest version of Tutor.
+
+This is clearly an area where contributions are needed. It will likely require one or more volunteers to help maintain
+master and fix breakages, as those would inevitably happen. Continuous deployment and some level of integration testing
+would also be required, as the volunteer(s) would need to be warned as soon as possible about failures.
+
+Close cooperation with the several engineering teams at edX will be needed: it would be best to have advance warning of
+backward-incompatible changes.
+
+
+Criterion 5. Customization
+..........................
+
+Tutor allows not only using `a custom Docker registry and images
+<https://docs.tutor.overhang.io/configuration.html#custom-images>`__, but also `building custom images
+<https://docs.tutor.overhang.io/configuration.html#custom-open-edx-docker-image>`__.  The latter is how one adds custom
+themes, xblocks, or even uses `a fork of edx-platform
+<https://docs.tutor.overhang.io/configuration.html#running-a-fork-of-edx-platform>`__.  In operational terms it is no
+harder than how one would accomplish the same with ``edx/configuration``.
+
+For many things that Tutor doesn't explicitly support (and for which there is no desire/need to include in the core), it
+is possible to `write plugins <https://docs.tutor.overhang.io/plugins/gettingstarted.html>`__.  With this engine, one
+can add additional services and configure the rest of the platform to use them. One can imagine writing plugins that
+allow deployment of the different Open edX MFEs, for instance.
+
+A note on theming: it would be ideal if it were possible to store static assets separately so that a single base image
+could be used for multiple instances, but that is not currently the case.
+
+
+Criterion 6. Migration from the native installation
+...................................................
+
+There is no easily found documentation on migrating to a Tutor-managed installation from a native one, aside from `this
+discussion in the Overhang.io forums <https://discuss.overhang.io/t/migrating-from-bitnami-to-tutor/31>`__.
+
+Assuming the source installation is running the same release as the target one, it would be a reasonably straightforward
+matter of launching a new Tutor stack (with all the desired customization applied, of course), dumping the state of the
+databases, then importing them.  Nevertheless, the exact steps need to be performed, tested, and written down.
+
+
+Criterion 7. Open edX development environment
+.............................................
+
+There is `specific documentation <https://docs.tutor.overhang.io/dev.html#development>`__ on using Tutor as an
+Open edX development environment.
+
+Support for master
+``````````````````
+
+Like the production version, it is not currently possible to run the ``master`` branch of ``edx-platform`` for
+development.
```

Comment:
Well... it _is_ posible, but it's almost certain that it will require some additional changes. It's probably not just a matter of running `tutor config save --set OPENEDX_COMMON_VERSION=master`. The probability that additional changes are required increases linearly with time since the last release. Thus, it is currently not officially supported.

Reply:
Good point.  I'll change "possible" to "supported".